src/connman.erl

```diff
@@ -134,6 +134,18 @@ disconnect(Tech, {name, Name}) ->
 disconnect(_Tech, {path, Path}) ->
     gen_server:call(?MODULE, {disconnect, Path}).
 
+%% @doc Remove the Wi-Fi service with the given name.
+-spec remove_wifi_service(string() | {service_key(), string()}) -> ok | {error, term()}.
+remove_wifi_service(Name) when is_list(Name) ->
+    remove_wifi_service({name, Name});
+remove_wifi_service({name, Name}) ->
+    case service_named("wifi", Name) of
```

Comment:
Can you make this match the other generic APIs that work with paths, tech and named services?

Reply:
> Can you make this match the other generic APIs that work with paths, tech and named services?

Yes, I can. I was on the fence about that since removing an `ethernet` service does not make a whole lot of sense but I agree that it's better to be consistent here.

Reply:
looks much better, ty!